sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslateContent.Serialization.cs

```diff
@@ -38,7 +38,7 @@ internal virtual MultipartFormDataRequestContent ToMultipartRequestContent()
             writer.WriteStartObject();
             writer.WritePropertyName("document"u8);
 #if NET6_0_OR_GREATER
-				writer.WriteRawValue(global::System.BinaryData.FromStream(Document));
+				writer.WriteRawValue(global::System.BinaryData.FromStream(MultipartDocument.Content));
```

Comment:
This will load the entire content stream into memory.  Are we confident that is safe to do?

Reply:
This is what is in the else statement, my intention was to match the behavior there. This #if was out of date b/c there was no net 6 or greater target, I was getting an error that "Document does not exist"